hw3/templates/register.html

```diff
@@ -0,0 +1,36 @@
+{% extends "base.html" %}
+{% block content %}
+<div align="center" class="col-10">
+<h1>Login</h1>
+<form method="POST" action="{{ url_for('register') }}">
```

Comment:
Атрибут `action` не нужен, так как этот шаблон контролируется функцией представления.
```suggestion
<form method="POST">
```